package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "aws-sqs-redriver",
+  "version": "1.0.0",
+  "description": "Re-drives messages from one SQS queue to another, e.g. to re-process \"dead letters\"",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint --cache .",
+    "build": "rm -rf lib && babel src --out-dir lib",
+    "prepublish": "npm run build",
+    "test": "NODE_ENV=test istanbul cover _mocha",
+    "test-no-cov": "NODE_ENV=test mocha",
+    "test-acceptance": "NODE_ENV=test istanbul cover _mocha -- --fgrep '/acceptance/'",
+    "test-component": "NODE_ENV=test istanbul cover _mocha -- --fgrep '/component/'",
+    "test-unit": "NODE_ENV=test istanbul cover _mocha -- --fgrep '/unit/'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Springworks/node-aws-sqs-redriver.git"
+  },
+  "keywords": [
+    "aws",
+    "sqs",
+    "redrive",
+    "lambda"
+  ],
+  "author": "Springworks <hello@springworks.se> (http://www.springworks.se)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Springworks/node-aws-sqs-redriver/issues"
+  },
+  "homepage": "https://github.com/Springworks/node-aws-sqs-redriver#readme",
+  "devDependencies": {
+    "@springworks/test-harness": "1.4.3",
+    "babel-cli": "6.14.0",
+    "babel-core": "6.14.0",
+    "babel-eslint": "6.1.2",
+    "babel-plugin-transform-runtime": "6.15.0",
+    "babel-plugin-transform-strict-mode": "6.11.3",
+    "babel-preset-es2015-node4": "2.1.0",
+    "eslint": "3.5.0",
+    "eslint-config-springworks": "7.0.6",
+    "eslint-plugin-import": "1.15.0",
+    "eslint-plugin-mocha": "4.5.1",
+    "eslint-plugin-should-promised": "1.0.8",
+    "eslint-plugin-springworks": "2.0.1",
+    "istanbul": "0.4.5",
```

Comment:
`next` is the name of the dist-tag in npm. It will install the pre-release version of 1.0